skaffold-setup.md

```diff
@@ -0,0 +1,29 @@
+# Using Skaffold for local dev
+
+## Install Skaffold
+
+See [here](https://skaffold.dev/docs/getting-started/#installing-skaffold)
+
+## Up and running
+
+Run
+
+```
+skaffold dev
+```
+
+## Notes / Gotchas
+- Skaffold will automatically detect the `docker-for-desktop` / `docker-desktop` contexts and not attempt to push an image. This can be overridden with:
+```
+build:
+  local:
+    push: false
+```
```

Comment:
where should this config be added (e.g. if this is being added to a file, what file?) to override the skaffold default?

Reply:
Can add this into the skaffold.yaml file directly